sql/shredder_state/progress/view.sql

```diff
@@ -0,0 +1,147 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shredder.shredder_state.progress`
+AS
+WITH max_end_date AS (
+  SELECT
+    MAX(end_date) AS end_date
+  FROM
+    `moz-fx-data-shredder.shredder_state.shredder_state`
+),
+shredder_state AS (
+  SELECT
+    task_id,
+    -- NEWEST job for each task_id
+    ARRAY_AGG(
+      STRUCT(
+        start_date,
+        end_date,
+        SPLIT(job_id, ":")[offset(0)] AS project_id,
+        SPLIT(job_id, ".")[offset(1)] AS job_id
+      )
+      ORDER BY
+        job_created DESC
+      LIMIT
+        1
+    )[OFFSET(0)].*
+  FROM
+    `moz-fx-data-shredder.shredder_state.shredder_state`
+  -- only for the most recent shredder run
+  JOIN
+    max_end_date
+  USING
+    (end_date)
+  GROUP BY
+    task_id
+),
+tasks AS (
+  SELECT
+    task_id,
+    ARRAY_AGG(
+      STRUCT(start_date, end_date, target, target_bytes, source_bytes)
+      -- already filtered on max_end_date, so use OLDEST metadata
+      ORDER BY
+        _PARTITIONTIME
+      LIMIT
+        1
+    )[OFFSET(0)].*
+  FROM
+    `moz-fx-data-shredder.shredder_state.tasks`
+  JOIN
+    max_end_date
+  USING
+    (end_date)
+  GROUP BY
+    task_id
+),
+jobs AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shredder.region-us.INFORMATION_SCHEMA.JOBS_BY_PROJECT`
+  UNION ALL
+  SELECT
+    *
+  FROM
+    `moz-fx-data-bq-batch-prod.region-us.INFORMATION_SCHEMA.JOBS_BY_PROJECT`
+),
+progress_by_task AS (
+  SELECT
+    task_id,
+    start_date,
+    end_date,
+    total_bytes_processed AS bytes_complete,
+    total_slot_ms AS slot_ms,
+  FROM
+    shredder_state
+  JOIN
+    jobs
+  USING
+    (project_id, job_id)
+  WHERE
+    creation_time > TIMESTAMP_SUB(CURRENT_TIMESTAMP, INTERVAL 28 DAY)
+    AND state = "DONE"
+    AND error_result IS NULL
+),
+progress_by_table AS (
```

Comment:
Is this the wrong label?

```suggestion
progress_by_target AS (
```